public/locales/en/demo.json

```diff
@@ -1,5 +1,6 @@
 {
   "app_config_not_found": "Invalid or Missing App Config",
+  "app_config_learn_more": "Learn more about the App Config creation",
```

Comment:
```suggestion
  "app_config_learn_more": "Learn more about the App Config",
```

Reply:
Will try to change, but the SEO score was lower because of this text
<img width="936" alt="Screenshot 2023-07-12 at 09 47 22" src="https://github.com/jwplayer/ott-web-app/assets/27254986/048b304e-eade-4106-8663-33712ef30c20">

Reply:
Looks like "Learn more" specifically is one of the generic flagged phrases: https://developer.chrome.com/en/docs/lighthouse/seo/link-text/